Assets/Scripts/EnemySpawn.cs

```diff
@@ -124,7 +124,7 @@ void SpawnEnemy()
         Vector3 targetSpawn;
         if (RandomPoint(platformRadius, out targetSpawn)) {
             Debug.DrawRay(targetSpawn, Vector3.up, Color.blue, 1.0f);
-            EnemyFactory.Instance.CreateEnemy(EnemyType.TURRET, targetSpawn);
+            EnemyFactory.Instance.CreateEnemy(targetSpawn, EnemyType.TURRET, EnemyVarientType.HEALER);

```

Comment:
correct me if i'm wrong, does this only spawn healer variants?

Reply:
Yeah im basically explicitly showing the new argument thats all